lib/bookmarks.rb

```diff
@@ -23,16 +23,23 @@
 require 'terminal-table/import'
 
 class Bookmarks
-  BOOKMARKS_PATH = File.expand_path "~/.jumprc"
-
+  @@BOOKMARKS_PATH = ""
   def initialize
+    rcfiles = [ "#{ENV["XDG_CONFIG_HOME"]}/jumprc", "~/.jumprc" ]
+    rcfiles.each do |_rcfile|
+      rcfile = File.expand_path _rcfile
+      if ( File.exists?(rcfile) )
+        @@BOOKMARKS_PATH = rcfile
+        break
+      end
+    end
     # Loads the bookmarks list from the bookmarks file.
     begin
-      @bookmarks = YAML::load_file(BOOKMARKS_PATH)
+      @bookmarks = YAML::load_file(@@BOOKMARKS_PATH)
     rescue Errno::ENOENT
       @bookmarks = {}
     rescue
-      raise "Can't load configuration file"
+      raise "Can't save configuration file #{@@BOOKMARKS_PATH}"
```

Comment:
Why did you change this message? This part of the code is actually loading the bookmarks.